### ISSUE TITLE
Unable to run tests

### ISSUE THREAD

**Issue by Heliosmaster** [ruhoh/ruhoh.rb#19]
Hi, I've forked the gem and tried to run the tests, but the output is this: http://pastebin.com/RTtDYuNm

**Comment by plusjade** [comment]
@Heliosmaster sorry about that. Seems I've hard coded the path to the sample blog directory in spec_helper.rb: https://github.com/ruhoh/ruhoh.rb/blob/0.2.0/spec/spec_helper.rb#L22 (this is on branch 0.2.0 which it looks like you are using)

You'll need to `git clone git://github.com/ruhoh/blog.git` the blog scaffold and then update the `SampleSitePath` to the location of the blog scaffold.

I know this isn't the best way to maintain the testing suite, but the API changes so frequently I don't want to have multiple versions of the blog scaffold floating around everywhere.

Basically all the failing tests are trying to test against an "actual blog" directory and ensure ruhoh is processing against the API correctly.

Hope this helps and let me know if you have any suggestions or improvements - thanks!

**Comment by Heliosmaster** [comment]
I was thinking about setting automated tests with travis, so I wanted first to run them on my machine. But if you have implemented it so, I am not sure it can be done with Travis.

**Comment by plusjade** [comment]
0.2.0 fixes this ( https://github.com/ruhoh/ruhoh.rb/pull/20) and integrates Travis as well with thanks to @ben-biddington. 0.2.0 is at HEAD now but not released on rubygems yet.
